src/main/java/io/vertx/core/net/NetServerOptions.java

```diff
@@ -274,8 +274,26 @@ public boolean isClientAuthRequired() {
    * @param clientAuthRequired  true if client auth is required
    * @return a reference to this, so the API can be used fluently
    */
+  @Deprecated
   public NetServerOptions setClientAuthRequired(boolean clientAuthRequired) {
-    this.clientAuthRequired = clientAuthRequired;
+    this.clientAuth = clientAuthRequired ? ClientAuth.REQUIRED : ClientAuth.NONE;
```

Comment:
I have got rid of internal one. Just so you know, I will still need to keep this here to keep the backward compatibility. Also, I have got rid of boolean flag "clientAuthRequired"